### DISCUSSION TITLE
Which package should I use?

### DISCUSSION THREAD

**Discussion by JamHadar** [LizardByte/community#272] [Sunshine] [answered]
It's probably silly to ask, but I'm new to Arch aur/flatpak and stuff and I can't find info on this.

Can you explain in short what the difference is between aarch64.flatpak, x86_64.flatpak and debug_x86_64.flatpak (the last one I'm pretty sure is not for me)?

Why do I ask: I tried installing via PKGBUILD. It runs, but Moonlight didn't see it. I tried installing via flatpak in terminal. I'm not sure, but I think installation borked at the end. I couldn't run it from terminal, only from app menu. It worked, but version was **.dirty. But still, Moonlight didn't see it.
Now I want to try to install it just with .flatpak and see what happens. And I'm also not sure if I need to make a search about permissions and stuff (and something about base-devel).

**Comment by ReenigneArcher** [comment]
`aarch64` means `arm64`, has nothing to do with Arch Linux.

Use `PKGBUILD`, `sunshine.pkg.tar.zst`, or use `sunshine` from AUR.

**Reply by JamHadar** [reply]
Thanks, I'll try it out

**Comment by KuleRucket** [comment, marked as answer]
I would only use the flatpaks if the AUR modules were not working but that's a personal choice. About the different AUR options:
sunshine - main releases
sunshine-git - nightly branch
sunshine-bin - downloads the zst from LizardByte releases and uses that.

The sunshine-bin is a good option as you don't need to wait for sunshine to build.

When you say you install via PKGBUILD, are you using a helper such as yay/paru? You should know how PKGBUILD installs work but a helper will make your life easier.

I have no idea why you have "dirty" packages. This isn't normal.

Finally, you can try manually adding the sunshine IP address to moonlight if it is not detected.

**Reply by JamHadar** [reply]
Yeah, the AUR option is on my "to try" list. Thanks for the info.

The "install via PKGBUILD" thing. I downloaded PKGBUILD from releases and used makepkg. I don't actually remember exactly how I installed it, I consumed too much info while researching, so itl's all kinda mixed up.

I tried adding host PC IP, but it didn't help either. Or, by saying sunshine IP address, you mean the sunshine itself has its own IP?
